suite-native/module-onboarding/src/components/GetTrezor.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { ReactNode } from 'react';
+
+import { Box, Stack, Text } from '@suite-native/atoms';
+import { Link } from '@suite-native/link';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+type GetTrezorProps = {
+    children: ReactNode;
+};
+
+const wrapperStyle = prepareNativeStyle(() => ({
+    width: '90%',
+}));
+
+export const GetTrezor = ({ children }: GetTrezorProps) => {
+    const { applyStyle } = useNativeStyles();
+
+    return (
+        <Stack spacing="large" style={applyStyle(wrapperStyle)}>
+            <Box flexDirection="row" alignItems="center" justifyContent="center">
+                <Text variant="hint" color="textSubdued">
+                    Don’t have a Trezor?
+                </Text>
+                <Link href="https://trezor.io/">
+                    <Text color="textPrimaryDefault"> Get one here.</Text>
+                </Link>
+            </Box>
+            {children}
```

Comment:
`children` prop is IMHO uncesary here. Whatever rendered here could be rendered next to this component one level above.

Reply:
[411b2ef](https://github.com/trezor/trezor-suite/pull/7889/commits/411b2ef4d48df39092779f8f118b3b7d57b00cc0)